composer.json

```diff
@@ -20,7 +20,8 @@
         "geshi/geshi": "^1.0.9",
         "setasign/fpdf": "^1.8",
         "paypal/rest-api-sdk-php": "^1.13",
-        "symfony/debug": "^3.4"
+        "symfony/debug": "^3.4",
+        "symfony/cache": "* v4.2.3"
```

Comment:
I am not sure what `"* v4.2.3"` means.
`*` means every version. `v4.2.3` means version 4.2.3.
I would be in favor of `^4.2` or `v4.2.3` (explicit defining and set it fixed to v4.2)

Reply:
Beats me, that is what Netbeans auto-inserted. Happy to change that.

Reply:
fixed as suggesteed